nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
```

Comment:
- [ ] Kindly make sure that the `Person` class inherits from `Nameable` class, then make sure that this class has a method `correct_name` implemented as per those [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Turn%20your%20Person,the%20name%20attribute.).